certora/specs/ghoVariableDebtToken.spec

```diff
@@ -6,60 +6,78 @@ import "summarizations.spec";
-	function _.getACLManager() external => CONSTANT;
-
-	/************************;
-    *    ACLManager.sol     *;
-    *************************/
-	function _.isPoolAdmin(address) external => CONSTANT;
-
-	/******************************************************************;
-	*	DummyERC20WithTimedBalanceOf.sol (linked to _discountToken)   *;
-	*******************************************************************/
-	// Internal function in DummyERC20WithTimedBalanceOf which exposes the block's timestamp and called by DummyERC20WithTimedBalanceOf::balanceOf
-	function _._balanceOfWithBlockTimestamp(address user, uint256 ts) internal => balanceOfDiscountTokenAtTimestamp(user, ts) expect uint256;
-
-  	/************************************;
-    *   DummyPool.sol (linked to POOL)  *;
-    *************************************/
-	// Internal function in DummyPool which exposes the block's timestamp and called by Pool::getReserveNormalizedVariableDebt
-	function _._getReserveNormalizedVariableDebtWithBlockTimestamp(address asset, uint256 timestamp) internal => indexAtTimestamp(timestamp) expect uint256;
-
-	/************************************;
-	*	GhoVariableDebtTokenHarness.sol	*;
-	*************************************/
-	function discStrategy.calculateDiscountRate(uint256, uint256) external returns (uint256) envfree;
-
-	/************************************;
-	*	GhoVariableDebtTokenHarness.sol	*;
-	*************************************/
-	function getUserCurrentIndex(address) external returns (uint256) envfree;
-	function getUserDiscountRate(address) external returns (uint256) envfree;
-	function getUserAccumulatedDebtInterest(address) external returns (uint256) envfree;
-	function getBalanceOfDiscountToken(address) external returns (uint256);
-
-	/********************************;
-	*	GhoVariableDebtToken.sol	*;
-	*********************************/
-	function totalSupply() external returns(uint256) envfree;
-	function balanceOf(address) external returns (uint256);
-	function mint(address, address, uint256, uint256) external returns (bool, uint256);
-	function burn(address ,uint256 ,uint256) external returns (uint256);
-	function scaledBalanceOf(address) external returns (uint256) envfree;
-	function getBalanceFromInterest(address) external returns (uint256) envfree;
-	function rebalanceUserDiscountPercent(address) external;
-	function updateDiscountDistribution(address ,address ,uint256 ,uint256 ,uint256) external;
-}
+    /********************;
+     *	WadRayMath.sol	*;
+     *********************/
+    //        function rayDiv(uint256 x, uint256 y) external returns uint256 envfree;
+    //function rayMul(uint256 x, uint256 y) external returns uint256 envfree;
+    //function _.rayMul(uint256 a,uint256 b) internal => rayMul_gst(a,b) expect uint256 ALL;
+    //function _.rayDiv(uint256 a,uint256 b) internal => rayDiv_gst(a,b) expect uint256 ALL;
+    //    function getIndex(uint256 bal) public view returns (uint256) 
```

Comment:
This is a problem if other rules uses this. you altered the summarization across all specs.
If you need this, potentially what you need is to open a new spec that inherit from this one and run it from there

Reply:
Commited by mistake...